MAINTAINERS

```diff
@@ -1,3 +1,15 @@
 Brian Exelbierd <bexelbie@redhat.com> @bexelbie
 Navid Shaikh <nshaikh@redhat.com> @navidshaikh
 Lalatendu Mohanty <lmohanty@redhat.com> @LalatenduMohanty
+
+## How to release the vagrant-service-manager plugin
+
+1. Put the gemfile in pkg/ with `rake build`
```

Comment:
or: "Create gem file under _pkg_ by running `rake build` ;-)
_put_ is misleading in this context indeed.